jest.config.js

```diff
@@ -1,6 +1,14 @@
 const path = require('path');
 const { compilerOptions } = require('./tsconfig.base.json');
+const { cwd } = require('process');
+
 const getTool = (name) => path.join(__dirname, 'tools', name);
+const getRoots = () => [
+  path.join(__dirname, '__mocks__'),
+  cwd().endsWith('TheCoin')
+    ? './'
+    : '<rootDir>/src'
+]
```

Comment:
As jest may either be run from root (via test explorer), or from an individual package, this jest config needs to adapt to either scenario with appropriate paths etc.